src/se/mod.rs

```diff
@@ -522,11 +523,18 @@ impl<'w, 'r, W: Write> Serializer<'w, 'r, W> {
                 level: QuoteLevel::Full,
                 indent: Indent::None,
                 write_indent: false,
+                expand_empty_elements: false,
             },
             root_tag: root_tag.map(|tag| XmlName::try_from(tag)).transpose()?,
         })
     }
 
+    /// Enable or disable expansion of empty elements.
+    pub fn expand_empty_elements(&mut self, expand: bool) -> &mut Self {
+        self.ser.expand_empty_elements = expand;
+        self
+    }
```

Comment:
Could you add a doctest example and specify that default value is `false`.

Reply:
@Mingun I added a dockest and noticed changing the expand_empty_elements in the method does not affect the `ContentSerializer` that actually does the serializing. 

I didn't find the culprit yet, is this due to macros or cloning?